src/assets/v3/ownership-transfer.njk

```diff
@@ -0,0 +1,28 @@
+---
+layout: layouts/base.njk
+---
+<div class="o-layout__sidebar"></div>
+
+<div class="o-layout__main o-layout-typography">
+
+<h1 id="new-chapter">Polyfill.io is entering a new chapter</h1>
+
+<p>A decade ago when polyfill.io started life, polyfills were vital to the advancement of web development practice. Today, the situation is quite different, and the time lag from a browser feature being firmly defined and it being widely supported is typically small, thanks to evergreen browsers. As a result, at the FT we aren't using polyfill.io any more, and new adoption of it has declined.</p>
+
+<p>It's time to think about what the future of polyfill.io looks like, since the FT is not going to be in a good position to be making substantial contributions in the future.</p>
+
+<p>We have therefore made the decision to transfer ownership of polyfill.io to the open source community. <a href="https://github.com/jakechampion">Jake Champion</a> who, at FT and Fastly, has been the most active contributor and maintainer for most of the polyfill.io’s life, will be taking on ownership of the service and related codebases, and will continue maintaining them with the input of other long-standing contributors. Hosting will continue to be provided free of charge by Fastly.</p>
+
+<h2 id="impact">What does this mean for you?</h2>
+
+<p>End users of the service should see no change. Transfer of ownership is largely an administrative and legal step, and the infrastructure and codebases will not be changing at all.</p>
```

Comment:
This is not true, the infrastructure will change, because the nameservers are going to be changing to ones I own and maintain. Also, the codebase will change as I will be maintaining and updating the codebase, it's not going to be frozen in time, it's going to be actively developed to make the best use of Fastly Compute@Edge etc

Reply:
Will the codebases remain under `https://github.com/Financial-Times` or will they be transferred to `https://github.com/jakechampion` or a new GitHub org?

It would be good to state explicitly what is moving and to where it will be moving.